RecoTracker/MkFitCore/standalone/Makefile.config

```diff
@@ -47,7 +47,7 @@ endif
 
 # 3. Optimization
 # -O3 implies vectorization and simd (but not AVX)
-OPT := -g -O3
+OPT := -g -Ofast
```

Comment:
There is a comment further down that should be moved ahead of this line and reworded like this:
# But use -Ofast rather than -O3 so gcc can auto-vectorize sin/cos with libmvec (glibc >= 2.22)

Note that eventually, all the icc stuff will have to be retooled for icx, as Intel is ending support for icc in 2H23.
However, such retooling may turn out to be minor, since icx is LLVM-based and accepts gcc-style options.

Reply:
Thank you, I did as suggested.